package.json

```diff
@@ -56,7 +56,7 @@
   },
   "dependencies": {
     "@atlas-viewer/iiif-image-api": "^2.1.1",
-    "@iiif/vault-helpers": "https://pkg.csb.dev/IIIF-Commons/vault-helpers/commit/8cc61732/@iiif/vault-helpers",
+    "@iiif/vault-helpers": "https://pkg.csb.dev/IIIF-Commons/vault-helpers/commit/a8d96591/@iiif/vault-helpers",
```

Comment:
`8cc6173` was a newer version (the netlify build is breaking due to missing export)